test/internal/pbxproj_partials/BUILD

```diff
@@ -0,0 +1,15 @@
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load(
+    ":write_pbxproject_prefix_tests.bzl",
+    "write_pbxproject_prefix_test_suite",
+)
+
+write_pbxproject_prefix_test_suite(name = "write_pbxproject_prefix")
+
+test_suite(name = "pbxproj_partials")
```

Comment:
I don't have experience with writing `@bazel_skylib//lib:unittest.bzl` tests tbh, mind clarifying why this `test_suite` rule is necessary here if it's not receiving any actual tests?

Reply:
If you don’t specify tests, it grabs all of them defined in the package. So it’s receiving the test suite listed above it.